ParisBNF/et/BNFet381.xml

```diff
@@ -109,6 +114,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <revisionDesc>
             
             <change who="ES" when="2024-04-22">created stub</change>
+            <change who="JK" when="2025-03-31">changed msIdentifier and xml:id of file</change>
```

Comment:
as always when renaming and moving - please add a formerlyAlsoListedAs relation, thank you very much